src/main/java/net/william278/velocitab/config/Placeholder.java

```diff
@@ -45,6 +45,10 @@ public enum Placeholder {
             .map(RegisteredServer::getPlayersConnected)
             .map(players -> Integer.toString(players.size()))
             .orElse("")),
+    LOCAL_GROUP_PLAYERS_ONLINE_((param, plugin, player) -> plugin.getTabGroups().getGroup(param)
+            .map(group -> Integer.toString(group.getPlayers(plugin).size()))
+            .orElse("Group " + param + " not found")),
+    LOCAL_GROUP_PLAYERS_ONLINE((plugin, player) -> Integer.toString(player.getGroup().getPlayers(plugin).size())),
```

Comment:
I don't think we need to distinguish "local" here &mdash; how about just `GROUP_PLAYERS_ONLINE`/`GROUP_PLAYERS_ONLINE_(name)`?